scalableminds/webknossos
# Store layer visibility with annotation

### URL of deployed dev instance (used for testing):
- https://layervisibilityinannotation.webknossos.xyz

### Steps to test:
- Create an annotation with multiple annotation layers
- toggle layer visibilities
- create second account, open the same annotation
- should show the layers the first user selected
- toggling by the second user should not save into the annotation

### Issues:
- fixes #5910

------
